t/op/sselect.t

```diff
@@ -106,6 +106,7 @@ select ($tie, undef, undef, $tie);
 ok("no crash from select $numeric_tie, undef, undef, $numeric_tie");
 
 SKIP: {
+    skip "Can't load modules under miniperl", 4, if is_miniperl;
```

Comment:
There’s a superfluous comma here after `4`.

I guess if Perl doesn’t complain about it, though, it’s no big deal.

Reply:
> There’s a superfluous comma here after `4`.
> 
> I guess if Perl doesn’t complain about it, though, it’s no big deal.

Thanks; will fix.